rllib/evaluation/multi_agent_sample_collector.py

```diff
@@ -0,0 +1,252 @@
+import logging
```

Comment:
This is the actual implementation of the SampleCollector API that will be used by RolloutWorker/Sampler to store and retrieve samples from env and models (then "packed" for inference calls, postprocessing batches and train batches).